intermediate_source/DCP_tutorial.rst

```diff
@@ -0,0 +1,238 @@
+Getting Started with Distributed Checkpoint (DCP)
+=====================================================
+
+**Author**: `Iris Zhang <https://github.com/wz337>`__, `Rodrigo Kumpera <https://github.com/kumpera>`__, `Chien-Chin Huang <https://github.com/fegin>`__
+
+.. note::
+   |edit| View and edit this tutorial in `github <https://github.com/pytorch/tutorials/blob/main/intermediate_source/DCP_tutorial.rst>`__.
+
+Checkpointing AI models during distributed training could be challenging, as parameters and gradients are partitioned across GPUs and the number of GPUs available could change when resume training.
+Pytorch Distributed Checkpointing (DCP) can help make this easier.
+
+In this tutorial, we show how to use DCP APIs with a simple FSDP wrapped model.
+
+
+How DCP works
+--------------
+torch.distributed.checkpoint enables saving and loading models from multiple ranks in parallel.
```

Comment:
```suggestion

:func:`torch.distributed.checkpoint` enables saving and loading models from multiple ranks in parallel.
```
you should be able to use [intersphinx](https://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html) here to point to the related sections in pytorch docs.